src/app/modules/main/communities/tokens/controller.nim

```diff
@@ -110,3 +130,12 @@ proc computeBurnFee*(self: Controller, contractUniqueKey: string, amount: Uint25
 
 proc getNetwork*(self:Controller, chainId: int): NetworkDto =
   self.networksService.getNetwork(chainId)
+
+proc getOwnerToken*(self: Controller, communityId: string): CommunityTokenDto =
+  return self.communityTokensService.getOwnerToken(communityId)
+
+proc getMasterToken*(self: Controller, communityId: string): CommunityTokenDto =
```

Comment:
We should call this `TokenMasterToken`. It sounds weird, but Master token is not a good name because it sounds like the Owner token.

Reply:
Or just `TMasterToken` :)

Reply:
Veto for TokenMasterToken. Why MasterToken is not good and OwnerToken is good?